lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -2799,3 +2799,22 @@ mapping:
           Allow disabling pbkdf2 hashing of passwords for legacy situations.
           This should normally be left enabled unless there is a specific
           reason to disable it.
+
+      use_multidomains:
```

Comment:
Maybe consolidate these two options and have the logic be 'if multidomains_root is set, use it'?  Then it's all handled by the single configuration param.

Reply:
Indeed, way more simple !